lib/kaffe/group_member/worker/worker.ex

```diff
@@ -0,0 +1,81 @@
+defmodule Kaffe.Worker do
+  @moduledoc """
+  A worker is assigned a single partition across topics. This is so we
+  effectively serialize the processing of any single key across topics.
```

Comment:
A subscriber is assigned to a partition within a topic, while a worker handles a partition for any topic within the client.

I'll clarify in the docs.